src/test/java/picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigarTester.java

```diff
@@ -228,7 +229,11 @@ public void test() {
         }
 
         // Also do tests from AbstractMarkDuplicatesCommandLineProgramTester
-        super.test();
+        try {
+            super.test();
```

Comment:
Pretty sure you just want the test to fail here, not succeed. Add `throws Exception` to this method to fix the compile error.